src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +185,114 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[TagsDict], session: Session) -> None:
+    session.add_tags(
+      ResourceArn=hub_content_arn,
+      Tags=tags
+    )
+    JUMPSTART_LOGGER.info(f"Added tags to HubContentArn %s: %s", hub_content_arn, TagsDict)
+    
+def find_unsupported_flags_for_hub_content_versions(hub_name: str, hub_content_name: str, region: str, session: Session) -> List[TagsDict]:
```

Comment:
unsupported is the wrong term to use here. i thought it means flags that were actually not supported, like some random string that is not something associated with JumpStart. can you rename `find_deprecated_vulnerable_flags_for_hub_content_versions` or something like that?

Reply:
Sounds good to me- that was my first idea for function names but wanted to consolidate to one word